portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,14 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
+  const facts =
+      ["I've lived in three different countries", "I'm a vegetarian", "My favorite Sunday activity is watching Formula 1", "I love cooking and doing yoga"];
```

Comment:
This line looks pretty long. You can also use a formatter for JavaScript too!

https://beautifier.io/